### DISCUSSION TITLE
Custom extension development guide

### DISCUSSION THREAD

**Discussion by aerfus** [dbeaver/dbeaver#12865] [Q&A]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version: CE edition 21.1.0
- Java version: OpenJDK 11.0.11
- Additional extensions: No

Hello.
I would like to create a custom extension to enhance queries with specific comments.
Is it possible to do that for DBeaver CE?
If yes, could you please provide me with a development guide?

The only note I have found so far is: https://dbeaver.io/forum/viewtopic.php?f=2&t=1836
but I'm not sure whether it is actual.

Thank you in advance.

**Comment by kseniiaguzeeva** [comment]
Could you please try this wiki article https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse? That's all what we have for now.
